src/components/about/Services.js

```diff
@@ -9,26 +9,26 @@ import { Col } from "react-bootstrap";
 
 const Services = () => {
   return (
-    <div className="bg-winc-pink flex flex-col w-full justify-center items-center p-8 outline outline-8 outline-winc-pink outline-offset-8 my-8  h-full">
-      <div className="flex justify-center pt-2 pb-16">
+    <div className="bg-winc-pink flex flex-col w-full justify-center items-center p-8 outline outline-8 outline-winc-pink outline-offset-8 my-8">
+      <div className="flex text-center justify-center pt-2 pb-16">
         <Title text="Become a Member" textcolor="text-winc-white" />
       </div>
-      <Row className="flex justify-center w-11/12 pb-16">
-        <Col className="flex flex-col items-center justify-center text-center">
+      <Row className="flex flex-col md:flex-row justify-center w-11/12 pb-16">
+        <Col className="flex flex-col py-2 items-center justify-center text-center">
```

Comment:
instead of py-2 everywhere, just change the service component to have a margin of my-2